www/feedback.js

```diff
@@ -20,6 +23,7 @@ function ctrl_key_name() {
     }  
 }
 
+if (document.addEventListener)
```

Comment:
You don't need to check for the document object?

Reply:
Yep, we define `document` as basically the very first thing in Chapter 9 (so we can define `document.querySelectorAll` so I figure it's OK to assume it exists.